src/stores/game.js

```diff
@@ -117,6 +116,13 @@ export const useGameStore = defineStore('game', () => {
   const gameHistoryStore = useGameHistoryStore();
 
   // Computed (getters)
+  const myPNum = computed(() => {
+    if (gameHistoryStore.isSpectating) {
+      return 0;
```

Comment:
@Mang0codes This will be the part to change in #1248 to read from the `pNum` query param and return the right value for `myPNum` when spectating